src/main/java/uk/gov/hmcts/ccd/v2/external/controller/CaseController.java

```diff
@@ -211,4 +219,43 @@ public CaseController(
 
         return status(HttpStatus.CREATED).body(new CaseResource(caseDetails, content, ignoreWarning));
     }
+
+
+    @GetMapping(
+        path = "/cases/{caseId}/events",
+        headers = {
+            V2.EXPERIMENTAL_HEADER
+        },
+        produces = {
+            V2.MediaType.CASE_EVENTS
+        }
+    )
+    @ApiOperation(
+        value = "Retrieve audit events by case ID",
+        notes = V2.EXPERIMENTAL_WARNING
+    )
+    @ApiResponses({
```

Comment:
Aren't we missing other response codes here like 422 UNPROCESSABLE_ENTITY from ValidationException?
Can you check whether others are not missing?

Reply:
added 422 messages to API